elex/cli/app.py

```diff
@@ -90,6 +90,11 @@ class Meta:
                 action='store',
                 help='Specify a value for a `batchname` column to append to each row.',
             )),
+            (['--officeids'], dict(
+                action='store',
```

Comment:
Why `store` instead of `store_true`? I haven't looked into what `action` does, would be curious.